ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+WKNavigationDelegate.swift

```diff
@@ -1831,7 +1831,7 @@ extension BrowserViewController: WKUIDelegate {
     }
 
     // Attempt to upgrade to HTTPS
-    if ShieldPreferences.httpsUpgradeLevel.isEnabled,
+    if FeatureList.kBraveHttpsByDefault.enabled, ShieldPreferences.httpsUpgradeLevel.isEnabled,
```

Comment:
Where do we fallback to our old way of upgrading to HTTPS i.e. via WebKit's internal mechanism?

Reply:
We do so when creating the web view here: https://github.com/brave/brave-core/blob/31c975b2ccc270c22e332f28d862029be5a44aa6/ios/brave-ios/Sources/Brave/Frontend/Browser/Tab.swift#L493

The above still matches the prior behaviour which was based on the old preference that looked like this:
```swift
configuration!.upgradeKnownHostsToHTTPS = Preferences.Shields.httpsEverywhere.value
```